apps/spruce/src/pages/task/taskTabs/TaskHistory/utils/utils.test.ts

```diff
@@ -7,14 +7,19 @@ import { getPrevPageCursor, getNextPageCursor, groupTasks } from ".";
 
 describe("groupTasks", () => {
   it("groups inactive tasks if shouldCollapse is true", () => {
-    const res = groupTasks(tasks, true);
+    const res = groupTasks(tasks, true, null);
     expect(res).toStrictEqual(collapsedGroupedTasks);
   });
 
   it("does not group inactive tasks if shouldCollapse is false", () => {
-    const res = groupTasks(tasks, false);
+    const res = groupTasks(tasks, false, null);
     expect(res).toStrictEqual(expandedGroupedTasks);
   });
+
+  it("sets isMatching to true if testFailureSearchTerm matches a failing test", () => {
+    const res = groupTasks(tasks, true, /e2e_test/);
+    expect("isMatching" in res[5] && res[5].isMatching).toBe(true);
```

Comment:
(after change from above)
```suggestion
    expect(res[5].isMatching).toBe(true);
```